paasta_tools/monitoring/check_capacity.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+# Copyright 2015-2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
```

Comment:
The pre-commit hooks are re-adding these when I commit: https://github.com/Yelp/paasta/blob/28c6d0cd59afc5aa45737a195af63213216aaae4/.pre-commit-config.yaml#L37-L40